pkgs/by-name/ch/chibi/package.nix

```
@@ -18,9 +18,9 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ makeWrapper ];
 
-  installPhase = ''
-    make install PREFIX="$out"
-  '';
+  makeFlags = [
+    "PREFIX=$(out)"
```

```suggestion
    "PREFIX=${placeholder "out"}"
```

This doesn't depend on shell

---

Thanks!